.github/workflows/CheckStdlibManifest.yml

```
@@ -0,0 +1,37 @@
+name: Stdlib Manifest is up-to-date
+
+on:
+  pull_request:
+    paths:
+      - 'stdlib/*.version'
+      - 'stdlib/*/Project.toml'
```

Presumably we want to also trigger this if `stdlib/Project.toml` is changed:

```suggestion
      - 'stdlib/*/Project.toml'
      - 'stdlib/Project.toml'
```